### ISSUE TITLE
URL mapping for "/" not working

### ISSUE THREAD

**Issue by jnunderwood** [jeffwils/grails-spring-security-saml#64]
This plugin is causing problems in my Grails 4.0.0 app. It seems to override the mapping for "/". It appears that your mapping is:

```groovy
"/"(view:"/index")
```

while my mapping is:

```groovy
"/"(controller: "home", action: "index")
```

But my SAML-enabled app keeps trying to go to "/index", which does not exist in my project. If I remove the SAML plugin from my project, everything works as expected. As soon as I include it, it stops working.

It _may_ be related to `grails-app/controllers/urlMappings.groovy`, but maybe not? Thanks for looking into this.

**Comment by valentingoebel** [comment]
UrlMappings.groovy definitively needs to be excluded from the plugin jar.

**Comment by valentingoebel** [comment]
4.0.1 has been released and should fix the problem.

**Comment by jnunderwood** [comment]
Thank you for the quick response. Version 4.0.1 seems to fix the problem :-)

**Comment by valentingoebel** [comment]
The pluginExcludes was wrong. I was getting class not found errors because the grails-plugin.xml was still pointing to the removed UrlMappings.groovy. I will fix this in 4.0.2.
